### ISSUE TITLE
no bugs message - maybe blank?

### ISSUE THREAD

**Issue by heckman** [heckman/SimPlant#11]
Instead of saying "No bugs to be seen." it could just say nothing at all.

**Comment by mowong** [comment]
Yeah, let's take it out, it gets annoying fast. I'll change it on my end.
